drivers/usb/device/usb_dc_stm32.c

```diff
@@ -303,6 +303,18 @@ static int usb_dc_stm32_init(void)
 	HAL_StatusTypeDef status;
 	unsigned int i;
 
+#ifdef SYSCFG_CFGR1_USB_IT_RMP
```

Comment:
Could we move it to `usb_dc_attach()` where we do also the pin remapping?

Reply:
Done